packages/webapp/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "prettier": "@dailydotdev/prettier-config",
   "dependencies": {
+    "@growthbook/growthbook": "https://gitpkg.now.sh/dailydotdev/growthbook/packages/sdk-js?a0d1a3add257acdbf5353dfd690547feebbada92",
```

Comment:
Was this intentional?

Reply:
Yes, related to https://github.com/dailydotdev/apps/pull/2337#discussion_r1386268280